src/components/DpEditor/DpEditor.vue

```diff
@@ -1064,6 +1064,9 @@ export default {
         this.emitValue()
       },
       editorProps: {
+        attributes: {
+          role: 'textbox'
+        },
```

Comment:
nitpick: formatting.

```suggestion
        },

```